src/mono/msbuild/common/LibraryBuilder.targets

```diff
@@ -32,6 +32,7 @@
       UsesCustomRuntimeInitCallback="$(_UsesCustomRuntimeInitCallback)"
       UsesRuntimeInitCallback="$(_UsesRuntimeInitCallback)">
       <Output TaskParameter="OutputPath" PropertyName="LibraryOutputPath" />
+      <Output Condition="'$(_IsSharedLibrary)' != 'true'" TaskParameter="ExportedSymbols" ItemName="LibraryRuntimeExportedSymbols" />
```

Comment:
Do we need to integrate `LibraryRuntimeExportedSymbols` into the current android/ios build? Would it need to be ingested here https://github.com/dotnet/runtime/blob/22b4f190b7d41dbe27e6c0a7cc96ef1619b7a36f/src/tasks/AndroidAppBuilder/ApkBuilder.cs#L256 ?

Reply:
In our case, we would duplicate what `NativeLib=shared` is doing, so I don't think it's necessary.